tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -163,32 +165,20 @@ def test__selection_config_from_dict() -> None:
     assert isinstance(cfg['strategies'][0]['input'], dict)
 
 
-def test__selection_config_from_dict__missing_n_samples() -> None:
-    cfg = {
-        "strategies": [
-            {"input": {"type": "EMBEDDINGS"}, "strategy": {"type": "DIVERSIFY"}},
-        ]
-    }
-    with pytest.raises(ValueError, match="Invalid value for `n_samples`, must not be `None`"):
-        api_workflow_compute_worker._selection_config_from_dict(cfg)
-
-
```

Comment:
This test no longer makes sense as `n_samples` is now optional

Reply:
Could we two tests instead that checks that both `n_samples` and `proportion_samples` can be set?

Reply:
Oh yes, great catch!

Reply:
It's actually verified by the `test__selection_config_from_dict` test.